composer/_version.py

```diff
@@ -3,4 +3,4 @@
 
 """The Composer Version."""
 
-__version__ = '0.25.0.dev0'
+__version__ = '0.26.0.dev0'
```

Comment:
this should be 
```suggestion
__version__ = '0.25.0'
```
so whne you use release branch (which will fork off this) its the right version and the tagged version is on this commit (which is on main branch)